defaults/main.yml

```diff
@@ -6,21 +6,32 @@ certbot_auto_renew_hour: "3"
 certbot_auto_renew_minute: "30"
 certbot_auto_renew_options: "--quiet --no-self-upgrade"
 
+certbot_testmode: false
+certbot_hsts: false
+
 # Parameters used when creating new Certbot certs.
 certbot_create_if_missing: false
 certbot_create_method: standalone
 certbot_admin_email: email@example.com
 certbot_certs: []
-# - email: janedoe@example.com
-#   domains:
-#     - example1.com
-#     - example2.com
-# - domains:
-#     - example3.com
+#   - email: janedoe@example.com
+#     webroot: "/var/www/html/"
+#     domains:
+#       - example1.com
+#       - example2.com
+#   - domains:
+#       - example3.com
+
 certbot_create_command: >-
-  {{ certbot_script }} certonly --standalone --noninteractive --agree-tos
+  {{ certbot_script }} certonly --{{ certbot_create_method  }}
+  {{ '--hsts' if certbot_hsts else '' }}
+  {{ '--test-cert' if certbot_testmode else '' }}
+  --noninteractive --agree-tos
   --email {{ cert_item.email | default(certbot_admin_email) }}
+  {{ '--webroot-path ' if certbot_create_method == 'webroot'  else '' }}
+  {{ cert_item.webroot if certbot_create_method == 'webroot'  else '' }}
```

Comment:
`cert_item.webroot | default(certbot_webroot)` could be useful in order to avoid to repeat `webroot` for each domain if it is common to all of them (in my case: `/var/www/letsencrypt`, not sure if this could be a good default value).

Reply:
Very good idea!